src/components/SurveyQuestion.js

```diff
@@ -0,0 +1,34 @@
+import classNames from "classnames";
+
+const SurveyQuestion = ({
+  question,
+  options,
+  selection,
+  onOptionClick,
+  onNextQuestion,
+}) => {
+  return (
+    <div>
+      <h1>{question}</h1>
+      <ul>
+        {options.map((option) => (
+          <li key={option.label}>
+            <button
+              className={classNames({
+                selected: selection.includes(option.label),
+              })}
+              onClick={() => onOptionClick(option.label)}
+            >
+              {option.label}
+            </button>
+          </li>
+        ))}
+      </ul>
+      <div>
+        <button onClick={onNextQuestion}>Next</button>
```

Comment:
Wait, this part was fine.
```suggestion
        <button onClick={onNextQuestion}>Next</button>
        <button className="prev" onClick={onPreviousQuestion}>
          Previous
        </button>
        {isLastQuestion ? (
          <button onClick={onFinish}>Finish</button>
        ) : (
          <button onClick={onNextQuestion}>Next</button>
        )}
```